File: past202004/past202004_b.cs
// https://atcoder.jp/contests/past202004-open/submissions/13444679

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        public static void Main()
        {
            var S = Console.ReadLine();
            Console.WriteLine(S.GroupBy(c => c).OrderByDescending(c => c.Count()).First().Key);
        }
    }
}

File: past202004/past202004_b.fs
// https://atcoder.jp/contests/past202004-open/submissions/13481596

stdin.ReadLine().ToCharArray()
|> Array.groupBy id
|> Array.sortByDescending (fun x -> (snd x).Length)
|> fun x -> fst x.[0]
|> stdout.WriteLine
